packages/docs/components/sidebar/Sidebar.vue

```diff
@@ -1,64 +1,130 @@
-          :active-by-default="item.name === route.name"
-          :to="item.name"
-        >
-          <span slot="title">{{ $t(item.displayName) }}</span>
-        </va-sidebar-link>
-      </template>
+      <va-list class="sidebar__links">
+        <va-expand-group v-model="value">
+          <va-expand
+            v-for="(item, key) in items"
+            :key="key"
+            class="sidebar__expand"
+            :class="{'sidebar__expand--active': value[key]}"
+          >
+            <template slot="header">
+              <va-list-item class="sidebar__category">
+                <va-list-item-section class="sidebar__category__section">
+                  <va-list-item-label class="sidebar__category__label">
+                    {{ $t(item.displayName) }}
+                  </va-list-item-label>
+                </va-list-item-section>
+                <va-list-item-section icon>
+                  <va-icon
+                    :name="value[key] ? 'expand_less' : 'expand_more'"
+                    :color="value[key] ? '#2c82e0' : ''"
+
+                  />
+                </va-list-item-section>
+              </va-list-item>
+            </template>
+            <div
+              v-for="(subMenuItem, index) in item.children"
+              :key="index"
+            >
+              <va-list-label
+                v-if="subMenuItem.category"
+                class="sidebar__subcategory__label"
+                color="gray"
+              >
+                {{ $t(subMenuItem.category) }}
+              </va-list-label>
+              <va-list-item
+
+                :to="subMenuItem.name"
+                class="sidebar__link"
+                active-class="sidebar__link--active"
+              >
+                <va-list-item-section>
+                  <va-list-item-label>
+                    {{ $t(subMenuItem.displayName) }}
+                  </va-list-item-label>
+                </va-list-item-section>
+              </va-list-item>
+            </div>
+          </va-expand>
+        </va-expand-group>
+      </va-list>
     </template>
   </va-sidebar>
 </template>
 
 <script lang="ts">
-// @ts-nocheck
-import { Component, Vue } from 'vue-property-decorator'
-import VaSidebar from './va-sidebar/VaSidebar.vue'
-import VaSidebarLink from './va-sidebar/VaSidebarLink.vue'
-import VaSidebarLinkGroup from './va-sidebar/VaSidebarLinkGroup.vue'
-import { navigationRoutes } from './NavigationRoutes.ts'
-@Component({
-  components: {
-    VaSidebarLinkGroup,
-    VaSidebarLink,
-    VaSidebar,
-  },
-  props: {
-    minimized: {
-      type: Boolean,
-      required: true,
-    },
-  },
-})
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { NavigationRoute } from './NavigationRoutes'
+
+@Component({})
 export default class Sidebar extends Vue {
-  data () {
-    return {
-      items: navigationRoutes.routes,
+  value = []
+
+  @Prop({ type: Array, default: [] }) readonly items!: NavigationRoute[]
```

Comment:
default: () => []

Your way won't work.

Also, any reason to have default here?